04_health/access-health-services/access-health-services-all.qmd

```diff
@@ -0,0 +1,829 @@
+    missing_pct = sum(is.na(ratio_population_pc_physician)) / n() * 100
+  )
+
+```
+
+
+
+### 6.3 Visualizations
+
+There are roughly the same number of observations with ratios over 10,000 in each year
+```{r check-outliers}
+# These outliers make the following histograms hard to interpret, so we check that there are roughly the same number in each year before filtering in the graphs below
+joined_data |>
+  filter(ratio_population_pc_physician >= 10000) |>
+  group_by(year) |>
+  count()
+
+```
+
+The distribution of ratios is roughly the same across all years
+```{r plot-distribution}
+joined_data |>
+  filter(!is.na(ratio_population_pc_physician),
+         # Filter out outliers, otherwise graphs are harder to interpret
+         ratio_population_pc_physician < 10000) |>
+  ggplot(aes(ratio_population_pc_physician)) +
+  geom_histogram(binwidth = 100) +
+  facet_wrap(~ year, scales = "free_x") +
+  labs(title = "")
+
+```
+
+
+### 6.4 Checking ratios against County Health Rankings
+The 2023 release of [County Health Rankings](https://www.countyhealthrankings.org/explore-health-rankings/county-health-rankings-model/health-factors/clinical-care/access-to-care/primary-care-physicians) data includes this same metric using the AHRF data. We check our calculations against theirs.
+
+For a while, we could not get our results to match the latest results from County Health Rankings. The number of primary care physicians were identical, but the ratios were not, which indicated that we used a different population source (population was not reported in CHR data). I recreated our ratio multiple times using different sources of population but could not match the CHR results.
+
+The CHR documentation indicates that they used 2021 Census Population Estimates for their ratio. However, on 2024-01-31, Vincent and Lily Robin met with a team of County Health Rankings researchers, and one of their researchers told us that they created their ratio using Vintage 2020 population estimates, which are based on the 2010 Census (see more [here](https://www.census.gov/programs-surveys/popest/technical-documentation/research/evaluation-estimates/2020-evaluation-estimates.html)). We are eventually able to recreate their ratio for 2020 using this population data. The ratio reported by County Health Rankings still does not match our ratio, but we used a more recent population count, so our ratio is likely more accurate (at least for 2020).
+
+#### 6.4a Download, read in, and clean the 2023 County Health Rankings data
+
+The landing page to download these data is [here](https://www.countyhealthrankings.org/explore-health-rankings/rankings-data-documentation).
+
+```{r load-chr}
+# Download file directly from internet
+download.file(
+  url = "https://www.countyhealthrankings.org/sites/default/files/media/document/2023%20County%20Health%20Rankings%20Data%20-%20v2.xlsx",
+  destfile = "04_health/access-health-services/data/chr2023.xlsx",
+  mode = "wb"
+)
```

Comment:
This didn't run for me because it's missing `here::here` in `destfile`. This fixes it:

```suggestion
 download.file(
  url = "https://www.countyhealthrankings.org/sites/default/files/media/document/2023%20County%20Health%20Rankings%20Data%20-%20v2.xlsx",
  destfile = here::here("04_health", "access-health-services", "data", "chr2023.xlsx"),
  mode = "wb"
)
```

Reply:
Fixed!